### PR TITLE
Grammar check

### PR DESCRIPTION
**Pull request by Ninja243**
This fixes multiple typos and refactors some sentences I found weird. :cowboy_hat_face:

### CODE REVIEW

File: docs/base/node_tutorial.md [hunk 1]
```diff
@@ -102,6 +102,20 @@ app.get('/users/:id', async (req, res) => {
...
+```js
+app.get('/search-by-age/:age, async (req, res) => {
+    const { age } = req.params;
+    const users = await db.fetch({'age': age});
+    if (users) {
+        res.json(user);
+    } else {
+        res.status(404).json({"message": "no users ${age} years old found"});
+    }
+});
```
Comment (abdelhai) [inline docs/base/node_tutorial.md]: I'm not sure about the code here but you don't need to check if it's empty, just return what the db gives you.
There are two possible response:

- Empty array
- Array with items

Both are acceptable response for the front end. No need to use `404`, 404 is only used when you look an item by a key IIRC.
```suggestion
    return (await db.fetch({'age': age}).next()).value;
});
```

File: docs/base/py_tutorial.md [hunk 2]
```diff
@@ -104,6 +104,18 @@ def get_user():
...
 
+We could also use the `db.fetch(query)` method to return the same data:
+
+
+```py
+@app.route("/users/<key>")
+def get_user():
+    key = request.json.get("key")
+    user = db_user.fetch({'key': id})
+    return user if user else jsonify({"error": "Not found"}, 404)
```
Comment (abdelhai) [inline docs/base/py_tutorial.md]: Same as the js one. 
```suggestion
    return list(db_user.fetch({'key': id}))
```

**Review by abdelhai — CHANGES_REQUESTED** [review]

File: docs/base/node_tutorial.md (hunk 1 above)
Comment (Ninja243) [reply docs/base/node_tutorial.md]: Makes sense to me! Good spot!

File: docs/base/py_tutorial.md (hunk 2 above)
Comment (Ninja243) [reply docs/base/py_tutorial.md]: :100:

**Review by abdelhai — APPROVED** [review]

**Pull request merged**